src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,11 @@
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todoes }) => (
+  <section className="TodoList">
+    {todoes && todoes.map(todo => (
```

Comment:
As I understand, your component will always receive `todos`, so this check unnecessary:  
```suggestion
    {todoes.map(todo => (
```